content/en/deployment_gates/setup.md

```diff
@@ -141,96 +174,50 @@ If a 200 HTTP status code is returned, the response is in the following format:
-exit 0
-```
+Note that the `deployment gate` command is available in datadog-ci versions v3.17.0 and above.
 
-The script has the following characteristics:
+**Required environment variables**:
+* `DD_API_KEY`: API key used to authenticate the requests.
+* `DD_APP_KEY`: Application key used to authenticate the requests.
+* `DD_BETA_COMMANDS_ENABLED=1`: The `deployment gate` command is a beta command, so datadog-ci needs to be ran with beta commands enabled.
 
-* It receives three inputs: `service`, `environment`, and `version` (optionally add `identifier` and `primary_tag` if needed). The `version` is only required if one or more APM Faulty Deployment Detection rules are evaluated.  
-* It sends a request to the Deployment Gate API and writes the output to the `response.txt` file.
-* It checks the HTTP response status code of the response, and does the following depending on the response code:
-  * 5xx: Retries the call (up to 3 times) with a delay of 5 seconds.
-  * Not 200 (for example, 404): Prints the resulting error and fails.
-  * 200: Checks the gate evaluation status returned (under `data.attributes.gate_status`) and passes or fails the script based on its value.  
-* If all the retries are exhausted (that is, several 5xx responses returned), the script does not return a failure to be resilient to API failures.
+For complete configuration options and detailed usage examples, refer to the [`deployment gate` command documentation][2].
 
-This is a general behavior, and you should change it based on your personal use case and preferences. The script uses `curl` (to perform the request) and `jq` (to process the returned JSON). If those commands are not available, install them at the beginning of the script (for example, by adding `apk add --no-cache curl jq`).
+[1]: https://github.com/DataDog/datadog-ci
+[2]: https://github.com/DataDog/datadog-ci/tree/master/src/commands/deployment#gate
 
-[1]: /getting_started/site/
 {{% /tab %}}
 {{% tab "Argo Rollouts" %}}
 To call Deployment Gates from an Argo Rollouts Kubernetes Resource, you can create an [AnalysisTemplate][1] or a [ClusterAnalysisTemplate][1]. The template should contain a Kubernetes job that is used to perform the analysis.
 
-Use this script as a starting point. For the API_URL variable, be sure to replace `<YOUR_DD_SITE>` with your [Datadog site name][3] (for example, {{< region-param key="dd_site" code="true" >}}).
+Use this script as a starting point. For the DD_SITE environment variable, be sure to replace `<YOUR_DD_SITE>` with your [Datadog site name][2] (for example, {{< region-param key="dd_site" code="true" >}}).
+
+The script needs an API and an Application key to be executed. The safest way to provide them is by using [Kubernetes Secrets][3]. This example relies on a secret called `datadog` holding two data: `api-key` and `app-key`. Alternatively, you can also pass the value in plain text using `value` instead of `valueFrom` in the script below.
```

Comment:
```suggestion
The script requires an API key and application key. The safest way to provide them is by using [Kubernetes Secrets][3]. This example relies on a secret called `datadog` holding two data values: `api-key` and `app-key`. Alternatively, you can also pass the values in plain text using `value` instead of `valueFrom` in the script below.
```